### PR TITLE
Feat

### PR DESCRIPTION
**Pull request by Michellebuchiokonicha**

### CODE REVIEW

File: control/src/pages/features/styles/Section1.module.css [hunk 1]
```diff
@@ -144,15 +151,30 @@
     justify-content: space-between;
     align-items: center;
   }
-  /* button {
+
+   .features {
+    width: 40%;
+   }
+   button {
```
Comment (ickynavigator) [inline control/src/pages/features/styles/Section1.module.css:158]: don't use non class selectors

File: control/src/pages/features/components/section3/Section3.js [hunk 2]
```diff
@@ -72,3 +72,4 @@ const Section3 = () => {
 }
 
 export default Section3
+hhh
```
Comment (ickynavigator) [inline control/src/pages/features/components/section3/Section3.js:75]: forgot to remove the text?

**Review by Michellebuchiokonicha — COMMENTED** [review]
There are a few changes made.

File: control/src/pages/features/components/section3/Section3.js (hunk 2 above)
Comment (Michellebuchiokonicha) [reply control/src/pages/features/components/section3/Section3.js:75]: which next?

Comment (Michellebuchiokonicha) [reply control/src/pages/features/components/section3/Section3.js:75]: you mean text

File: control/src/pages/features/styles/Section1.module.css (hunk 1 above)
Comment (Michellebuchiokonicha) [reply control/src/pages/features/styles/Section1.module.css:158]: owh

**Pull request closed without merging**